index.html

```diff
@@ -315,9 +315,93 @@ <h2 class="component_ex">Example of Floating action Button</h2>
                     </iframe>
                 </div>
                 <hr>
+                <!-- Image component -->
+                <div id="Image">
+                    <h1 class="component_heading">Image</h1>
```

Comment:
Same as above. As this may create confusions while implementing js selecting elements.